clang/unittests/Support/TimeProfilerTest.cpp

```diff
@@ -60,13 +75,29 @@ bool compileFromString(StringRef Code, StringRef Standard, StringRef FileName) {
   return Compiler.ExecuteAction(Action);
 }
 
+std::string GetMetadata(json::Object *Event) {
+  std::string Metadata;
+  llvm::raw_string_ostream OS(Metadata);
+  if (json::Object *Args = Event->getObject("args")) {
+    if (auto Detail = Args->getString("detail"))
+      OS << Detail;
+    if (auto File = Args->getString("file"))
+      OS << ", "
+         << llvm::sys::path::filename(File->str(),
```

Comment:
Could you add a comment that explains that we do this to make the tests outputs the same between Windows and Linux?

Reply:
Done.